coordinator-common/src/main/java/org/apache/kafka/coordinator/common/runtime/CoordinatorRuntime.java

```diff
@@ -196,6 +197,11 @@ public Builder<S, U> withExecutorService(ExecutorService executorService) {
             return this;
         }
 
+        public Builder<S, U> withMaxBufferSizeSupplier(Supplier<Integer> maxBufferSizeSupplier) {
+            this.appendMaxBufferSizeSupplier = maxBufferSizeSupplier;
```

Comment:
nit: Could we name this `withAppendMaxBufferSizeSupplier`? for consistency
```suggestion
        public Builder<S, U> withAppendMaxBufferSizeSupplier(Supplier<Integer> appendMaxBufferSizeSupplier) {
            this.appendMaxBufferSizeSupplier = appendMaxBufferSizeSupplier;
```